app/models/event.rb

```diff
@@ -41,6 +41,7 @@ class Event < ActiveRecord::Base
   scope :canceled, -> { where(state: 'canceled') }
   scope :withdrawn, -> { where(state: 'withdrawn') }
   scope :highlighted, -> { where(is_highlight: true) }
+  scope :scheduled_events, -> { confirmed.select(&:scheduled?) }
```

Comment:
Now you are not using this scope, so you can remove it. Sorry for the confusion, I realised that it was better to use the method in program afterwards. :confounded: